apps/docs/content/tokens/getting-started/introduction.mdx

```diff
@@ -8,41 +8,72 @@ order: 1
 
 At the heart of Workleap Design language are tokens, which drive its functionality. They are the key ingredients to everything UI. Workleap brand is dissected into design tokens, they live in the Hopper Design System.
 
+A design token can store any of the following elements:
+
+- A single value such as a `color`, a `font-size`, a `border-width`.
+- A collection of indivisible values such as a `box-shadow`.
+
+## Definitions
+
+Hopper aims to provide a three tier token system, each tiers have their specific purposes.
+
 <Card>
     <Image src="https://assets.workleap.com/hopper/webdoc/token-details.png" width="654" height="45" alt="Token tiers diagram" />
 </Card>
 
-## Definitions
+### Core tokens
+
+These tokens are referring to raw values such as an hexadecimal color code, a border-width, a font-size. They are the building blocks of the design system.
+
+<Table category="color" data={[{
+    "name": "hop-sapphire-200",
+    "value": "#95b1ff"
+}]} />
+
+### Semantic tokens
+
+Adds the intention layer to tokens, they map to a core token, or tokens should we say as the semantic tier has a notion of themability. They are the tokens that are used in the components.
+
+#### Light
+
+<Table category="color" data={[{
+    "name": "hop-primary-surface-disabled",
+    "value": "#95b1ff"
+}]} />
+
+#### Dark
+
+<Table category="color" data={[{
+    "name": "hop-primary-surface-disabled",
+    "value": "#2040c7"
+}]} />
+
+### Component tokens
+
+Refers to the nuts and bolts of the components and their implementation details, they are the tokens exposed by components. They are tied to their respective component. _Note: As no components are currently available these tokens are not yet available._
```

Comment:
"they are the tokens exposed by components" should either be a new sentence of separated by a semi colon.

```suggestion
Refers to the nuts and bolts of the components and their implementation details; they are the tokens exposed by components. They are tied to their respective component. _Note: As no components are currently available, these tokens are not yet available._
```